viewer/viewer-awt/src/main/java/org/icepdf/ri/common/print/PrintHelperFactoryImpl.java

```diff
@@ -0,0 +1,19 @@
+package org.icepdf.ri.common.print;
+
+import org.icepdf.core.pobjects.PageTree;
+
+import javax.print.attribute.DocAttributeSet;
+import javax.print.attribute.PrintRequestAttributeSet;
+import java.awt.*;
+
+public class PrintHelperFactoryImpl implements PrintHelperFactory {
+
+    public static final PrintHelperFactoryImpl INSTANCE = new PrintHelperFactoryImpl();
```

Comment:
For API consistency can you convert this to a getInance() method and make the constructor private.